generators/add-action/audience-manager-cd/index.js

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+const { commonDependencyVersions } = require('../../../lib/constants')
+
+class AudienceManagerCDGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {
+      description: 'This is a sample action showcasing how to access the Adobe Audience Manager Customer Data API',
+      // eslint-disable-next-line quotes
+      requiredParams: `['apiKey', 'orgId']`,
```

Comment:
awesome
according to the last changes of orgId from requiredparams to requiredheaders
i will have to change the stub-action.js as well to pass my tests, but in this case the other action templates will failed because of the existing orgId as a paramters too
i will push the changes after your [pull request](https://github.com/adobe/generator-aio-app/pull/44) been pushed

Reply:
Hi @chiarabdy good point thanks for pointing this out, the PR just got merged now :)